### PR TITLE
Add RADIS to OpenAstronomy

### PR DESCRIPTION
**Pull request by erwanp**
Following discussion with @dpshelio , i create this PR. Please let me know once the new member guidelines are updated. 

RADIS is a package for infrared molecular spectroscopy. The package is gaining interest from people working on exoplanet detection (wherever they use ExoMol or other large line databases). I'd like to push towards that direction and joining OpenAstronomy is a first step.

### CODE REVIEW

**Comment by erwanp** [comment]
Hello @dpshelio ! I'd like to suggest one or two small projects for GsOC 2020 as part of OpenAstronomy. Do you think RADIS has a chance to be included before that?

**Comment by dpshelio** [comment]
@erwanp I believe so. We are planning to have a call soon to discuss the memberships. I would suggest you work on your GSoC ideas and open a PR for them as soon as you have a draft, so we can help you to get it in better shape. The deadline for the ideas is not the same as for OA to apply to participate, so we have some time after that to gather and make the ideas better.
If in the worst scenario we can't accept RADIS, then you can use the same ideas and try with the Python Software Foundation.

File: _data/members.yaml [hunk 1]
```diff
@@ -265,6 +265,22 @@ ctlearn:
    chat:
      slack: https://ctlearn.slack.com
 
+radis:
+   name: radis
+   logo: radis_ico.png # name of the logo as saved on img/members
+   url: radis.readthedocs.io/
```
Comment (dpshelio) [inline _data/members.yaml]:
```suggestion
   url: https://radis.readthedocs.io/
```

**Comment by mirca** [comment]
@dpshelio @bsipocz  I guess this is good to go now?

**Comment by bsipocz** [comment]
@mirca - that is also pending on the OA committee meeting.

File: _data/members.yaml [hunk 2]
```diff
@@ -265,6 +265,22 @@ ctlearn:
    chat:
      slack: https://ctlearn.slack.com
 
+radis:
+   name: radis
+   logo: radis_ico.png # name of the logo as saved on img/members
+   url: https://radis.readthedocs.io/
+   repositories:
+     github: https://github.com/radis/radis
```
Comment (dpshelio) [inline _data/members.yaml]:
```suggestion
     github: radis/radis
```

**Comment by bsipocz** [comment]
Thanks @erwanp!

**Pull request merged**